frontend/app/module/data_entry/PollingStation/PollingStation.test.tsx

```diff
@@ -20,10 +20,15 @@ const startPollingStationInput = async () => {
   expect(router.state.location.pathname).toEqual("/elections/1/data-entry/1");
 };
 
-const expectRecountedForm = async () => {
+const expectRecountedForm = async (headingShouldNotHaveFocus?: boolean) => {
```

Comment:
I personally have something against negative boolean names, especially when used in an inverse way :)
Maybe we can do `async (headingShouldHaveFocus = true) {` instead and flip all usages?

Reply:
Changed in 8dc28d65183b3e99fae58abe59646b139e4ae842